src/driftpy/drift_client.py

```diff
@@ -72,7 +72,7 @@ def __init__(
         self.subaccounts = [0]
 
         if account_subscriber is None:
-            account_subscriber = CachedDriftClientAccountSubscriber(self.program)
+            account_subscriber = WebsocketDriftClientAccountSubscriber(self.program)
```

Comment:
this is default to websocket? is that same as typescript

Reply:
yeah